content/case_histories/bookpurnong/properties.rst

```diff
@@ -3,3 +3,25 @@
 Properties
 ==========
 
+The sandy topsoil at Bookpurnong is mostly saturated by water from the river, irrigation, and other source of recharge. The ground conductivity is therefore dominated by the salinity of groundwater. High salinity in water means more ions acting as free charge carriers, so high electrical conductivity is expected at the place suffers from high salinization (see more information under Physical Properties > Electrical Conductivity). At Bookpurnong, the fresh water is from the river and irrigation, and the saline water is from the natural acquifer under the highland. Therefore, the conductivity can be a direct indicator of the source of groundwater, from the river or the intrusion. 
+
+For the purpose of studying the effect of SIS, hydrogeologists have sampled and measured the electrical conductivity of the groundwater along several transects across the river floodplain (Figure :numref:`booky-drills`). The measured conductivity from several wells along Transect 3 is plotted as cross sections from the river bed to the bank both before and after SIS (Figure :numref:`booky-salinity`). 
+
+
+.. figure:: ./images/booky-drills.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-drills
+
+.. figure:: ./images/booky-salinity.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-salinity
```

Comment:
On my laptop this figure is larger than the screen. Do you have the same effect?